library/package.json

```diff
@@ -40,8 +40,7 @@
     "build:styles:dev": "cross-env NODE_ENV=production postcss src/styles/default.css -o styles/default.css --verbose",
     "build:styles:prod": "cross-env NODE_ENV=production MINIFY_STYLES=true postcss src/styles/default.css -o styles/default.min.css --verbose",
     "test": "npm run test:unit",
-    "test:unit": "jest --env=jsdom --detectOpenHandles",
-    "test:unit:watch": "jest --detectOpenHandles --watch",
+    "test:unit": "jest --env=jsdom --detectOpenHandles .",
```

Comment:
You can define `env` for `jest` in the `jest.config.js`

Reply:
Lets fix it later yea 🙂